content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,184 @@
+======================
+Requests for quotation
+======================
+
+.. _inventory/purchase/manage_deals/request_for_quotation:
+
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |POs| replace:: :abbr:`POs (Purchase Orders)`
+.. |RFQ| replace:: :abbr:`RFQ (Request for Quotation)`
+.. |RFQs| replace:: :abbr:`RFQs (Requests for Quotation)`
+
+Odoo’s requests for quotation (RFQs) feature in the **Purchase** app standardizes
+ordering products from multiple vendors with varying prices and delivery times.
```

Comment:
Remove curly quote (`’`), these pesky quotes can be carried over from copy/pasting from a Google doc or other formatted text editor.

Also there is an early line break in this sentence, can break the line closer to 100 chars :)
```suggestion
Odoo's requests for quotation (RFQs) feature in the **Purchase** app standardizes ordering products
from multiple vendors with varying prices and delivery times.
```